.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Changelist
+<!-- Bulleted list of changes made -->
+- 
+
+## Breaking Changelist
+<!-- Delete section if none, bulleted list of breaking changes -->
```

Comment:
IMO this shouldn't be deleted so that reviewers are also reminded to keep this in mind.
```suggestion
<!-- Bulleted list of breaking changes; write "none" if no breaking changes -->
```